src/ml/neural_net/mps_layer_instance_norm_data_loader.h

```diff
@@ -0,0 +1,92 @@
+#ifndef mps_layer_instance_norm_data_loader_h
+#define mps_layer_instance_norm_data_loader_h
+
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h> 
+
+typedef struct{
```

Comment:
We should avoid defining types in the global namespace with generic names. I don't see this being used anywhere anyway, so can we just delete this one?